community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -502,11 +512,18 @@ private static String defaultPageCacheMemory()
     @Internal
     public static final Setting<String> auth_manager = setting( "unsupported.dbms.security.auth_manager", STRING, "" );
 
-    // Bolt Settings
+    @Description("Default network interface to listen for incoming connections. " +
+            "To listen for connections on all interfaces, use \"0.0.0.0\". " +
+            "To bind specific connectors to a specific network interfaces, " +
+            "specify the +listen_address+ properties for the specific connector.")
+    public static final Setting<String> default_listen_address =
+            setting( "dbms.connectors.default_listen_address", STRING, "localhost" );
 
-    @Description("Hostname/IP address and port that we can be connected to be the driver.")
-    public static final Setting<HostnamePort> bolt_advertised_address =
-            setting( "bolt_advertised_address", HOSTNAME_PORT, "localhost:7687" );
+    @Description("Default hostname or IP address the server uses advertise itself its connectors. " +
+            "To advertise a specific hostname or IP address for a specific connector, " +
+            "specify the +advertised_address+ property for the specific connector.")
+    public static final Setting<String> default_advertised_hostname =
```

Comment:
Should it be `default_advertised_address` here, as it is in the `@Description` and in the connector-specific settings?